Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelNavigatableListItem.swift

```diff
@@ -57,7 +60,10 @@ public struct ChatChannelNavigatableListItem<Factory: ViewFactory, ChannelDestin
                 tag: channel.channelSelectionInfo,
                 selection: $selectedChannel
             ) {
-                LazyView(channelDestination(channel.channelSelectionInfo))
+                LazyView(
+                    channelDestination(channel.channelSelectionInfo)
+                        .modifier(HideTabBarModifier(handleTabBarVisibility: handleTabBarVisibility))
```

Comment:
We already have handling for the tab bar, somewhere, this would duplicate the handling. Maybe we just optionally add it between 16.0 and 16.3?

Reply:
This modifier is being used on Thread List, I'm reusing this one. The other one that we have is simpler and does not work on every scenario, and it is on the Demo App only

Reply:
You need to test this with <iOS 16 simulator. If that works, I'm fine with removing all that code.